utility/types/message.go

```diff
@@ -40,6 +42,8 @@ type Message interface {
 
 	SetSigner(signer []byte)
 	ValidateBasic() Error
+	// Get the canonical byte representation of the ProtoMsg.
+	GetSignBytes() []byte
```

Comment:
`Signed` makes me thing "signed message". Is that what's intended here?

Reply:
Right, as far as I remember I was inspired by V0 also on naming. 

It should be "get the bytes that I have to sign".

`GetBytes` seems too generic and technically we are marshalling the message and returning the `[]byte` (and potentially we might sort the JSON as well in there)

How about `GetCanonicalBytes`?

Reply:
`GetSignableBytes`?

Reply:
I like `GetCanonicalBytes`